virttest/utils_libvirt/libvirt_unprivileged.py

```diff
@@ -26,6 +26,7 @@ def get_unprivileged_vm(vm_name, user, passwd, **args):
     params = utils_params.Params()
     params['connect_uri'] = f'qemu+ssh://{user}@localhost/session'
     params['serials'] = args.get('serials', 'serial0')
+    params['status_test_command'] = 'echo $?'
```

Comment:
Couldn't we reuse `virttest/shared/cfg/guest-os/Linux.cfg:5:    status_test_command = echo $?`

Reply:
ok